tutorials/docs-00-getting-started/00_getting-started.jmd

```diff
@@ -0,0 +1,102 @@
+---
+title: Getting Started
+permalink: /docs/using-turing/get-started
```

Comment:
Will this `permalink` work when we use version control? If not, is there a way to insert the `version` (e.g. https://turing.ml/v0.21/tutorials/00-introduction/) to permalinks in Jekyll?

Summon our god of Jekyll @cpfiffer

Reply:
Yes, I think it should go through just fine because the root includes the version number.